pubsub/pubsub.go

```diff
@@ -74,11 +74,6 @@
 //    ever call Message.Ack.
 //  - func() { log.Info("ack called!") }: log. Softer than panicking.
 //
```

Comment:
Sorry, please only delete the last sentence from the paragraph, not the whole paragraph.

Reply:
Oh yes. It was indeed clear, sorry about that...